tc/core/halide2isl.h

```diff
@@ -82,10 +83,8 @@ struct ScheduleTreeAndAccesses {
   /// for each leaf node is captured below.
   tc::polyhedral::ScheduleTreeUPtr tree;
 
-  /// Union maps describing the reads and writes done. Uses the ids in
-  /// the schedule tree to denote the containing Stmt, and tags each
-  /// access with a unique reference id of the form __tc_ref_N.
-  isl::union_map reads, writes;
+  /// Information extracted from the bodies of the statements.
```

Comment:
it makes sense to rename the enclosing struct from `ScheduleTreeAndAccesses` to something else, it has outgrown that name

Reply:
On Wed, Jun 27, 2018 at 09:55:01AM -0700, ftynse wrote:
> it makes sense to rename the enclosing struct from `ScheduleTreeAndAccesses` t
o something else, it has outgrown that name

To what?  "halide2isl::Scop"?

skimo